openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateAvailabilityManagerNightCycle.cpp

```diff
@@ -66,17 +66,22 @@ boost::optional<IdfObject> ForwardTranslator::translateAvailabilityManagerNightC
   }
 
   {
-    auto schedule = modelObject.model().alwaysOnDiscreteSchedule();
+    auto schedule = modelObject.applicabilitySchedule();
     idfObject.setString(AvailabilityManager_NightCycleFields::ApplicabilityScheduleName,schedule.name().get());
   }
 
-  if( airLoopHVAC ) {
-    // Fan schedules are set to match the availabilitySchedule in the translator
-    idfObject.setString(AvailabilityManager_NightCycleFields::FanScheduleName,airLoopHVAC->availabilitySchedule().name().get());
-  }
 
-  // TODO: @kbenne, we don't even translate the AVM:NightCycle if it's not on an airloop anyways
+  // Note: always true, we don't even translate the AVM:NightCycle if it's not on an airloop anyways
```

Comment:
Ok good point. Maybe when I wrote that comment I was thinking about the future. I think now in energyplus the AVMs can be applied to zone equipment (can't they?), but if we ever add support in OS we can address the complications then.

Reply:
The comment was mine, it was a TODO I left in a PR when I extended AVM night cycle to add the controp zones field etc. But yeah, the if block was yours and your analysis is spot on: when we start supporting them for zonehvac (I filled an enhancement request for this a few weeks ago) this reminder will be useful.

Reply:
https://github.com/NREL/OpenStudio/pull/2869#pullrequestreview-78721850